configuration/arm64/initialization.sh

```diff
@@ -11,3 +38,12 @@ sudo apt-get -y install docker-ce docker-ce-cli containerd.io docker-compose-plu
 sudo apt -y install python3-pip
 sudo pip3 install docker-compose
 sudo docker-compose up -d
+
+#create v2xhub user
+cd mysql
+./add_v2xhub_user.bash
+
+sudo apt-get update -y && sudo apt update -y
+
+echo -e "\n open chromium-browser, go to https://127.0.0.1:19760, accept certs, and go to https://127.0.0.1 to login to V2x hub."
+sleep 3
```

Comment:
Is this sleep necessary.

Reply:
will remove when updating to have chrome tabs remove automatically

Reply:
updated both scripts to open 2 chrome tabs, one to localhost and other to localhost:19760 to accept certs instead of prompting user to do so.